java/core/src/java/org/apache/orc/impl/ColumnStatisticsImpl.java

```diff
@@ -35,13 +35,8 @@
 import org.apache.orc.TimestampColumnStatistics;
 import org.apache.orc.TypeDescription;
 
-import java.nio.ByteBuffer;
-import java.nio.CharBuffer;
-import java.nio.charset.Charset;
-import java.nio.charset.StandardCharsets;
 import java.sql.Date;
 import java.sql.Timestamp;
-import java.util.Arrays;
```

Comment:
Shall we exclude these unrelated changes? It makes backports difficult sometimes.

Reply:
Good point, but do we need to carry on with un-used imports for upcoming releases, they can accumulate over time. Also, for backporting it could be  relatively easy to add these dependencies (I am assuming manual backports). Let me know if you feel strongly about it.

Reply:
We had better split that kind of cosmetic commits and critical bug fix commits.

Reply:
@dongjoon-hyun I reverted these changes.